torch/optim/lr_scheduler.py

```diff
@@ -1268,15 +1268,20 @@ def __init__(self,
 
         self.cycle_momentum = cycle_momentum
         if cycle_momentum:
-            if 'momentum' not in optimizer.defaults:
-                raise ValueError('optimizer must support momentum with `cycle_momentum` option enabled')
+            if 'momentum' not in optimizer.defaults and 'betas' not in optimizer.defaults:
+                raise ValueError('optimizer must support momentum or beta1 with `cycle_momentum` option enabled')
```

Comment:
Can we make this line consistent with whatever is in OneCycleLR either by changing that one as well or by leaving this the previous?

Reply:
I believe they are in sync? See 1722 in OneCycleLR for the matching error.  Maybe we want it to be betas instead of beta1? Adam or betas based optimizers were supported in OneCycleLR but I wanted it to be clear in the error message.

Reply:
Right now, the error for the OneCycleLR is: 
raise ValueError('optimizer must support momentum with `cycle_momentum` option enabled')

without the "or beta1". I am okay with the addition of or beta1, but I think both these messages should be the same.

Reply:
https://github.com/pytorch/pytorch/pull/113548/files#diff-036a7470d5307f13c9a6a51c3a65dd014f00ca02f476c545488cd856bea9bcf2R1732

I'm confused because I have it changed in this PR. Maybe it's somewhere else?

Reply:
oh sorry! I must have missed it--thank you for the pointer